test/runtests.jl

```diff
@@ -190,7 +189,6 @@ cd(dirname(@__FILE__)) do
         println("    \033[32;1mSUCCESS\033[0m")
     else
         println("    \033[31;1mFAILURE\033[0m")
-        Base.Test.print_test_errors(o_ts)
-        error()
+        exit(-1)
```

Comment:
we probably don't want including this file from the repl to exit the calling process